include/assimp/MemoryIOWrapper.h

```diff
@@ -80,7 +80,8 @@ class MemoryIOStream : public IOStream
     // -------------------------------------------------------------------
     // Read from stream
     size_t Read(void* pvBuffer, size_t pSize, size_t pCount)    {
-        const size_t cnt = std::min(pCount,(length-pos)/pSize),ofs = pSize*cnt;
+        ai_assert(pvBuffer && pSize);
```

Comment:
Please do not check two statements in one asserrt. Which one will trigger the assert?

Reply:
Both conditions must always pass, otherwise we have undefined behavior (division by zero for pSize and write access to nullptr for pvBuffer). It's just a sanity test.
```suggestion
        ai_assert(pvBuffer);
        ai_assert(pSize);
```